atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/CellsSweeper.java

```diff
@@ -90,6 +94,13 @@ public void sweepCells(
                     sentinelsToAdd);
         }
 
+        if (cellTsPairsToSweep.entries().stream().anyMatch(entry -> entry.getValue() == null)) {
+            log.error("When sweeping table {} found cells to sweep with the start timestamp null."
+                            + " This is unexpected. The cellTs pairs to sweep were: {}.",
+                    SafeArg.of("tableRef", LoggingArgs.tableRef(tableRef)),
```

Comment:
Don't rewrap the `Arg` from `tableRef` as this will potential convert an `UnsafeArg` into a `SafeArg`.